CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Each line is a file pattern followed by one or more owners.
+# Owners will be requested for review when someone opens a pull request.
+
+# Fallback owner.
+# These are the default owners for everything in the repo, unless a later match
+# takes precedence.
+* @frequenz-floss/api-team
```

Comment:
We usually have also the SDK team as an owner for the `py/` and `pytest/` directories, you can see other API repos as an example. But do it in a separate PR, so we have a clean import.